src/index.html

```diff
@@ -11,6 +11,35 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post"
+    data-qa="big"
+      >
+      <label class="search" for="bigSearch"
+      >
+        <input
```

Comment:
![image](https://user-images.githubusercontent.com/99652389/178430975-a9dc655d-4f1e-49e3-9e71-f52877dd81d6.png)
from `checklist.md` in the root folder of  homework.